dataproc/README.md

```diff
@@ -20,49 +20,78 @@ Go to the [Google Cloud Console](https://console.cloud.google.com).
 Under API Manager, search for the Google Cloud Dataproc API and enable it.
 
 
-# Set Up Your Local Dev Environment
+## Set Up Your Local Dev Environment
+
 To install, run the following commands. If you want to use  [virtualenv](https://virtualenv.readthedocs.org/en/latest/)
 (recommended), run the commands within a virtualenv.
 
     * pip install -r requirements.txt
 
 Create local credentials by running the following command and following the oauth2 flow:
 
-    gcloud beta auth application-default login
+    gcloud auth application-default login
```

Comment:
Our current auth guidance is to use service accounts.

Reply:
Took this out, linked to our auth docs, noted we recommend service accounts.